mlrun/feature_store/feature_set.py

```diff
@@ -1030,6 +1030,61 @@ def deploy_ingestion_service(
             self, source, targets, name, run_config, verbose
         )
 
+    def extract_relation_keys(
+        self,
+        other_feature_set,
+        relation: Dict[str, Union[str, Entity]] = None,
+    ) -> list[str]:
+        """
+        This method checks if the other feature set can be merged to the right of this feature set
+
+        :param other_feature_set:   The feature set you want to merge to the right of this feature set
+        :param relation:            All the relation that where defined on this feature set.
+
+        :return:                    If those 2 feature set are merge able it returns a list of the left join keys
+                                    otherwise it returns an empty list.
+                                    (The right join keys are always the entities of the other feature set)
+        """
```

Comment:
```suggestion
        relations: Dict[str, Union[str, Entity]] = None,
    ) -> list[str]:
        """
        Checks whether a feature set can be merged to the right of this feature set.

        :param other_feature_set:   The feature set to be merged to the right of this feature set.
        :param relations:            The relations that were defined on this feature set.

        :return:                    If the two feature sets can be merged, a list of the left join keys is returned.
                                    Otherwise an empty list is returned.
                                    (The right join keys are always the entities of the other feature set)
```

Reply:
Can also keep it singular ("relation"), but then please be consistent with that.